harness/determined/cli/model.py

```diff
@@ -224,6 +229,14 @@ def register_version(args: Namespace) -> None:
                     Arg("--json", action="store_true", help="print as JSON"),
                 ],
             ),
+            Cmd(
+                "delete",
+                delete,
+                "delete model",
+                [
+                    Arg("name", type=str, help="unique name of the model"),
+                ],
```

Comment:
I am not sure if it'd be useful at all to have a `--json` arg detailing the deleted model, but I'm going to assume not :D